addons/purchase/report/purchase_order_templates.xml

```diff
@@ -135,4 +135,7 @@
         </t>
     </t>
 </template>
+
+<template id="document_tax_totals" inherit_id="account.document_tax_totals_template" primary="True"></template>
```

Comment:
You can take off the `primary=True` here too then ^^

Reply:
Edit: see https://github.com/odoo/odoo/pull/106140#discussion_r1081495797
We kept the primary in the end